### ISSUE TITLE
SGD momentum in args but Adam Optimizer in code ( which only use learning rate)

### ISSUE THREAD

**Issue by hazebroufe** [hayashimasa/UNet-PyTorch#4]
Hey, in the training code (train.py), the lines 51 to 54 :

    parser.add_argument(
        '--momentum', type=float, default=0.5, metavar='M',
        help='SGD momentum (default: 0.5)'
    )

are useless and misleading as the code uses Adam optimizer, and only feeds it the learning rate as argument.

-> FIX: just remove the lines
